test/unit/Models/account.spec.ts

```diff
@@ -0,0 +1,96 @@
+import {CGAccount} from "../../../src/renderer/models/account";
+import {ICGKeystore, ICGKeystoreFactory} from "../../../src/renderer/services/interfaces";
+import {Keypair} from "@chainsafe/bls/lib/keypair";
+
+// Passwords for keystores 1 & 2
+const PRIMARY_KEYSTORE_PASSWORD = "chainGuardianPass";
+
+/*eslint-disable*/
+const CGKeystoreTest: ICGKeystoreFactory = class CGKeystoreTest implements ICGKeystore {
+    private file: string;
+    constructor(file: string) {
+        this.file = file;
+    }
+
+    decrypt(password: string): Keypair {
+        if (password === PRIMARY_KEYSTORE_PASSWORD) {
+            return Keypair.generate();
+        } else {
+            throw new Error("Incorrect password");
+        }
+    }
+    // Methods bellow are not important for Account class
+    changePassword(oldPassword: string, newPassword: string): void {
+        throw new Error("Method not implemented.");
+    }
+    destroy(): void {
+        throw new Error("Method not implemented.");
+    }
+    create(file: string, password: string, keypair: Keypair): ICGKeystore {
+        throw new Error("Method not implemented.");
+    }
+};
+/*eslint-enable*/
+
+function createTestAccount(): CGAccount {
+    return new CGAccount("Test Account", "./test/test_keystores/", false, CGKeystoreTest);
```

Comment:
you should not use real keystore implementation. You keystore should be stubbed with sinon so you can simulate errors and stuff and we won't have to wait for decryption  in unit tests